#Servo Control State

#Constants:

#Servos:
ServoState Servo1
ServoState Servo2
ServoState Servo3
ServoState Servo4
ServoState Servo5
ServoState Servo6
ServoState Servo7
ServoState Servo8
